### DISCUSSION TITLE
Ctrl p/n navigation

### DISCUSSION THREAD

**Discussion by zhixian001** [johnlindquist/kit#238] [Ideas]
It would be great if kit supports `ctrl+p` (↑) or `ctrl+n` (↓) for script navigation!

In addition to the arrow keys or text searching, I wish there were more ways to choose scripts.

**Comment by johnlindquist** [comment]
I'll look into more ways of selecting choices from the input. I need to keep in mind the prompt often "generates" choices (from apis, etc) whereas a ctrl+p/n requires choices to exist to know what is prev/next match.
